### DISCUSSION TITLE
Budget with multiple currencies

### DISCUSSION THREAD

**Discussion by guisil** [moneymanagerex/moneymanagerex#4152] [Q&A]
Greetings,

I just started exploring the Budget feature. I created a budget for the current year, but not all transactions are appearing there. I have multiple accounts, most in Euros, but one of them in a different currency and it happens to be the one I use the most. Only the transactions of the accounts in Euros are shown in the budget.
Is there something I should do beforehand to enable this or is this simply not implemented yet?

**Comment by whalley** [comment]
I think it is an issue in the budget setup view of actuals. The budget reports seem to correctly represent the actual spend across all accounts irrespective of currency.

Please raise a bug report and we can look into it.

**Reply by guisil** [reply]
> I think it is an issue in the budget setup view of actuals. The budget reports seem to correctly represent the actual spend across all accounts irrespective of currency.

Actually this is not the case for me. The reports also don't seem to show anything besides the transactions in Euros.
I see that for every report I open, the "Currency:" field in the header is always empty. Is this normal?

**Comment by guisil** [comment]
Ok, I was trying to replicate this in a newly created database before creating a bug report, and it turns out it's working. 
After looking more carefully at what might be causing the problem I suspect it's the currencies themselves.

If I go to the "Organize Currency" dialog in my newly created database, it contains a large list of currencies. And, after I download the latest rates, my secondary currency (PLN) has an updated value.

<img width="801" alt="new_database" src="https://user-images.githubusercontent.com/15089071/153754526-7d3133c2-b57c-49e4-8ba9-61c987be4e81.png">

In my old database, however, the list of currencies is much smaller and the rate is never updated. It always shows an "Invalid value" error dialog for PLN. It shows the same error also if I change the value manually to be larger than zero.

<img width="802" alt="old_database" src="https://user-images.githubusercontent.com/15089071/153754546-1d340faf-3cee-4d9a-a29a-182ed9504855.png">

<img width="799" alt="error" src="https://user-images.githubusercontent.com/15089071/153754833-90d70f38-1279-497f-8234-c4b0d9ee1842.png">

Is there a way to update or somehow import this list of currencies?

**Reply by vomikan** [reply]
What is your base currency?

**Reply by guisil** [reply]
EURO, the second one in the small list.

**Reply by vomikan** [reply]
> EURO, the second one in the small list.

It should have EUR symbol and marked (the symbol like V or flag in the first column) as base currency.

**Reply by guisil** [reply]
Ok, I added "EUR" as the currency symbol and now it is updating the rates. Also, the budget now includes everything.
Thank you!
